Made changes based off of Elliot's comments

kaygomo opened the pull request:
Centered the header image, made body narrower, and padded sections.

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
```
Move this CSS into a separate style.css file.

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            <div class="padded">
+            <!-- see if you can get this a little bit narrower so that the text wraps to more lines to match the design a little closer -->
+            
+            <p style="font-family:Helvetica;">
```
Refrain from using inline styles. If you're going to style a paragraph, give it a class or ID and put it in your stylesheet with a declaration. Alternatively, if Helvetica is the global font, define it somewhere like `body {}`

```
.title {
 font-family: helvetica;
}
```

```
body {
     font-family: helvetica !default;
     font-size: 12px !default;
}
```

Read: https://css-tricks.com/using-default-parameters-es6/

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+
+            <div id="rcorners4"><img src="michelangelo.gif" alt="Michelangelo" class="rounded-corners" style="width:200px;height:200px;">
+            </div>
+
```
Maintain consistency with your classes and IDs and explicitly name them so the reader knows what they are.
.container-round {}.

If all of these container have rounded corners, share a class instead of using an ID. IDs are for single case usage.

Read: https://teamtreehouse.com/library/dry-css

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            <div id="rcorners1"><img src="raphael.gif" alt="Raphael" class="rounded-corners" style="width:200px;height:200px;">
+            </div>
+
+            <span class="name4"><b>Raphael</b></span>
```
If possible, don't use bold, italics, or underline elements. Always define these in the CSS. if needed, make a special case for the container. This will allow you to change formatting/layout site wide if needed.

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            wears a purple mask and wields a bo staff. Donatellow is perhaps the least
+            violent turtle, preferring to use his knowledge to solve conflicts, but 
+            never hesitates to defend his brothers. 
+            <br>
```
Can you avoid using breaks tags by changing the container's padding or margin?

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name2"><b>Donatello</b></span>
+            <hr style="height:0px;border-top: 3px solid plum" />
```
Can you make a custom class to share this style with all hr tags?

```
hr.something {
    height: 0;
    padding-bottom: 100px;
    border-top: #DDA0DD;
}
```

`<hr class="something">`

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+  display: flex;
+  justify-content: center;
+}
+             
```
When writing your CSS follow the standard brace style.

```
selector { 
    property: value; /* Start Properties on Newline */
    property: value;
}
```

Read: https://google.github.io/styleguide/htmlcssguide.html

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+<html>
```
Do you need to declare a doctype?

zachshallbetter replied:
- This document should be name index.html so it's the default page that loads when visiting this folder

Read: https://teamtreehouse.com/community/why-is-it-important-that-we-name-the-main-file-indexhtml

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
```
All of these name elements share the same formatting. You should separate them.

I explain the DRY approach further into this document.

.red-title {
    color: #FFFFF
}

.name-text {
     font-size: 30px;
     font-family: Helvetica;
}

<span class="name-text red-title">text</span>

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
```
Try not to create generic selectors like `.padded`. Down the road, you might find yourself using variations of padding.

```
.container-something {
     padding: 50px 400px 400px;
}
```

Use shorthand whenever possible.

` padding: 50px 400px 50px 400px;`

Read: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            margin-left: 20px
+                    }          
+
+                    .main {
```
`Main` is typically used for wrapping container, and even then should be avoided. Use` #main-content` or something similar.

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+border-radius: 15px;
+}
+
+             .image-container {
```
This is used appropriately, but make sure you understand flex-box and the box model.

Read: https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Box_model
Play: https://flexboxfroggy.com/
Read: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox;

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+        <body>
+                
+                <!-- get the header image to center on the page -->
+                <div class="image-container"><img src="TMNTHead.png" alt="TMNTHead" style="width:1400px;height:832px;"></div>
```
- Alt tags should be descriptive so that the visually impaired know what they're reading.
- width and height should be in the css
- images should be in their own folder `/img`

`<img src-"" alt="Teenage Mutant Ninja Turtles Heads">`

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            <br>
+            <br>
+             He is named after the Renaissance painter, sculptor, architect, poet,
+            and engineer, <a href="https://en.wikipedia.org/wiki/Michelangelo" style="color:orange">Michelangelo.</a></span></div>
```
This link should have a target so it doesn't open in the same window.

`<a href="" target="_Blank"></a>`

zachshallbetter commented on Mulligan.1.html:
```diff
@@ -0,0 +1,174 @@
+            <div id="rcorners2"><img src="leonardo.gif" alt="Leonardo" class="rounded-corners" style="float:left;width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name"><b>Leonardo</b></span>
```
This span should be a heading  element"`<h2>`"so it follows a semantic HTML approach for improved readability, accessibility, and ADA requirements.

Read: http://accessiblehtmlheadings.com/
Read: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines

kaygomo commented:
1. Created a separate style sheet
2. Modified alt tags
3. Modified links w/ "target"
4. Declared doc type/renamed doc to index.html
5. Created classes for image containers, hr tags, and other styling elements
6. Got rid of those damn <br> tags

kaygomo commented:
Added title
Created image directory and add relative links
Modified various class names/using semantic class descriptions (ie. pagepadding to page-padding and rcorners2 to rounded-container-corners)
Got rid of all inline css in the html doc
Styled spans, paragraphs, etc
Changed mich to Mike because I'm not from Boston
Removed hr class
Used hex values

kaygomo commented:
Oh toots, I missed two colors. I just fixed them, once I edit the doc again I'll re-commit
